SPDX-FileCopyrightText: © 2021 Andrius Štikonas <andrius@stikonas.eu>

SPDX-License-Identifier: GPL-2.0-or-later

Fixes aclocal to work with our Perl

--- bin/aclocal.in  2021-03-31 18:19:50.665806225 +0100
+++ bin/aclocal.in  2021-03-31 18:20:02.836132739 +0100
@@ -42,6 +42,7 @@
 use Automake::XFile;
 use Automake::FileUtils;
 use File::Basename;
+use File::Glob;
 use File::Path ();
 
 # Some globals.
